### PR TITLE
Added user group dams-editor and remove admin group dams-rci. DM-186

### PR DESCRIPTION
**Pull request by lsitu**
https://lib-jira.ucsd.edu:8443/browse/DM-186

### CODE REVIEW

**Comment by lsitu** [comment]
@mcritchlow Have you got a chance to review this PR? We may have to remove the unit dependency for access control now. And I have a PR for damspas which dependences on this decision as well. Thanks.

File: src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java [hunk 1]
```diff
@@ -898,20 +898,9 @@ private void outputTransform( String objid, String cmpid, String fileid,
...
-				}
-				else
-				{
-					// use default admin group2 if none specified
-					params.put("adminGroup", new String[]{roleAdmin});
-					params.put("adminGroup2", new String[]{roleAdmin2});
-				}
+				String adminGroup = roleAdmin;
+				params.put("adminGroup", new String[]{adminGroup});
+				params.put("adminGroup2", new String[]{roleAdmin2});
```
Comment (mcritchlow) [inline src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java:902]: where is `roleAdmin2` defined?

Comment (lsitu) [inline src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java:902]: It's configured in dams.properties: https://lib-stash.ucsd.edu/projects/ND/repos/private_config/browse/damsrepo/production/dams.properties#58 .

Comment (mcritchlow) [inline src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java:902]: Ah, I see. Thanks.

I think I was thrown off by the naming convention. Do we need the `adminGroup` String instance?  Or can we just use existing `roleAdmin` variable instead?

Comment (lsitu) [inline src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java:902]: I've erased that line and use variable `roleAdmin` directly. Thanks.

**Comment by mcritchlow** [comment]
👍

**Pull request merged**